### DISCUSSION TITLE
After upgrade 0.11..0.12 has problem with withPackage extension

### DISCUSSION THREAD

**Discussion by JajaComp** [LemonAppDev/konsist#535] [Report a Problem] [answered]
### Tell us what version of Konsist you use.

0.12

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Duplicate functions withPackage/withoutPackage in same package:
Overload resolution ambiguity: 
public fun <T : KoResideInOrOutsidePackageProvider> List<TypeVariable(T)>.withPackage(vararg names: String): List<TypeVariable(T)> defined in com.lemonappdev.konsist.api.ext.list
public fun <T : KoResideInPackageProvider> List<TypeVariable(T)>.withPackage(vararg names: String): List<TypeVariable(T)> defined in com.lemonappdev.konsist.api.ext.list

### Actual behaviour

Sample

### Expected behaviour

Sample

**Comment by igorwojda** [comment, marked as answer]
Thx for reporting 🙏
Hot fix has been just released a hotfix https://github.com/LemonAppDev/konsist/releases/tag/v0.12.1
